Week3/homework/4-step3.js

```diff
@@ -1,2 +1,10 @@
 'use strict';
 // paste your freeCodeCamp solutions in here
+
+function Dog() {
+    this.name = "Lili";
+    this.color = "white";
+    this.numLegs = 4;
+}
+
+const hound = new Dog();
```

Comment:
This would work just fine but only in case when we need just one instance of a dog. What happens if I need multiple dogs? You can achieve that scenario a little differently like this:
```
function Dog(name, color, legs) {
    this.name = name;
    this.color = color;
    this.numLegs = legs;
}

// new Dog("Lili", "white", 4);
// new Dog("Don", "black", 4);
```
Makes sense?